sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeRequestBanner.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <banner
+    class="mb-2"
+    :type="BannerTypes.Warning"
+    :header="bannerText"
+    :summary="bannerSummary"
+  />
+</template>
+<script lang="ts">
+import { ref, defineComponent } from "vue";
+import { BannerTypes } from "@/types/contracts/Banner";
+export default defineComponent({
+  setup() {
+    const bannerText = ref("This request is still pending with the student");
+    const bannerSummary = ref(
```

Comment:
These texts are static. So why do they need to be a ref?

Reply:
This component is now removed.